src/controllers/authenticationController.ts

```diff
@@ -224,11 +234,20 @@ export default class AuthenticationController {
       this.lollipopParams.isLollipopEnabled &&
       O.isSome(errorOrMaybeAssertionRef.right)
     ) {
+      const assertionRefToRevoke = errorOrMaybeAssertionRef.right.value;
       // Sending a revoke message for previous assertionRef related to the same fiscalCode
       // This operation is fire and forget
       this.lollipopParams.lollipopService
-        .revokePreviousAssertionRef(errorOrMaybeAssertionRef.right.value)
+        .revokePreviousAssertionRef(assertionRefToRevoke)
         .catch(err => {
+          this.appInsightsTelemetryClient?.trackEvent({
+            name: "lollipop.error.acs",
+            properties: {
+              assertion_ref: assertionRefToRevoke,
+              error: err,
+              fiscal_code: sha256(user.fiscal_code)
+            }
+          });
           log.error(
             "acs: error sending revoke message for previous assertionRef [%s]",
             err
```

Comment:
Prepend a message error that allow to discriminate what `lollipop.error.acs` event is this one.
We can reuse the log.error one.

Reply:
I added a message property, I think it's more clean. https://github.com/pagopa/io-backend/pull/996/commits/275e94a174fed76bf20736cb6040e924ad01a415